vci-service-impl/src/main/java/io/mosip/certify/vci/filter/AccessTokenValidationFilter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.vci.filter;
+
+import io.mosip.certify.core.constants.Constants;
+import io.mosip.certify.core.dto.vci.ParsedAccessToken;
+import io.mosip.certify.core.util.IdentityProviderUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
+import org.springframework.security.oauth2.jwt.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+
+
+@Slf4j
+@Component
+public class AccessTokenValidationFilter extends OncePerRequestFilter {
+
+    @Value("${mosip.certify.vci.authn.issuer-uri}")
```

Comment:
should we remove .vci. since certify is vc issuance

Reply:
removed .vci from the properties